DM-50006: Add a section on Piff and an intro on PSFex/Piff. (#30)
lsst-pst/pstn-019: meas_psf.tex

\subsection{meas\_extensions\_psf\_package}

> Within the pipeline, two distinct PSF models are used: \texttt{PSFex} and \texttt{Piff}. \texttt{PSFex} is 

Reviewer: If we are talking about PSFs in general, then there's a PCA version too that we just don't use it in the pipelines. We should make this sound as if we talk about two of the PSF models, or specify the rest (my preference is for the former)

Author: Oh yes, let me add a precision on this.

> the initial, fast, and less accurate PSF estimation and is wrapped within \texttt{meas\_extensions\_psfex}. 

Reviewer: ```suggestion
a fast, and less accurate PSF estimation and is wrapped within \texttt{meas\_extensions\_psfex}. 
```

Reviewer: Running it initially is a specific choice we have made for our pipeline. As a software, there's no requirement that this be the initial one at all.

> In contrast, \texttt{Piff} is the subsequent, slower, but more accurate PSF estimation that is incorporated 

Reviewer: ```suggestion
In contrast, \texttt{Piff} is a slightly slower, but more accurate PSF estimation that is incorporated 
```
